### ISSUE TITLE
Request: Download as pdf

### ISSUE THREAD

**Issue by thibaultmol** [lhunath/bash.academy#9]
Hi

I'd be great if there was an option somewhere on the website to download the entire thing as a PDF 'book'. Kind off like how you can generate a PDF of a wikipedia page: [https://en.wikipedia.org/wiki/Help:Download_as_PDF#Creating_a_PDF](https://en.wikipedia.org/wiki/Help:Download_as_PDF#Creating_a_PDF)

Thanks
Thibault
Greeting from Flanders btw :smile:

**Comment by lhunath** [comment]
Agreed, but it's a little harder on guide.bash.academy since it's not backed by a Wiki engine.  But definitely a good thought.

Groetjes,
Maarten.

**Comment by perpi** [comment]
Are you serious?! Won't be there a pdf version for now? Oh common, man; A guide should be something on papers! preferably available in epub/mobi format for ebook-readers.

**Comment by GreenLunar** [comment]
Download as ePUB!
